sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample4_MultivariateDetect.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing.Text;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Net.NetworkInformation;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.AI.AnomalyDetector.Models;
+using Azure.Core.TestFramework;
+using Microsoft.Identity.Client;
+using NUnit.Framework;
+
+namespace Azure.AI.AnomalyDetector.Tests.Samples
+{
+    public partial class AnomalyDetectorSamples : SamplesBase<AnomalyDetectorTestEnvironment>
```

Comment:
Can we add an md file for this sample with the description like other three samples [here](https://github.com/Azure/azure-sdk-for-net/tree/7c71881e7406c63a17de67c7b52ee1be67648d9d/sdk/anomalydetector/Azure.AI.AnomalyDetector/samples)?

Reply:
I still don't see an md file [here](https://github.com/Azure/azure-sdk-for-net/tree/7c71881e7406c63a17de67c7b52ee1be67648d9d/sdk/anomalydetector/Azure.AI.AnomalyDetector/samples) for `Sample4_MultivariateDetect.cs` sample. We need to create a `Sample4_MultivariateDetect.md` file like [this](https://github.com/Azure/azure-sdk-for-net/blob/7c71881e7406c63a17de67c7b52ee1be67648d9d/sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample1_DetectEntireSeriesAnomaly.md) with the description explaining the sample.